static/app/stores/selectedGroupStore.tsx

```diff
@@ -48,12 +43,11 @@ const storeConfig: SelectedGroupStoreDefinition = {
   },
 
   reset() {
-    this.records = new Map();
-    this.lastSelected = null;
+    this.state = {records: new Map(), lastSelected: null};
   },
 
   getState() {
-    return Object.fromEntries(this.records);
+    return this.state;
```

Comment:
Oh it's probably this returning different data, the issue stream component using it needs to be changed